Engine/plugin/plugin_stubs.cpp

```diff
@@ -62,11 +62,12 @@ bool RegisterPluginStubs(const char* name)
     ccAddExternalStaticFunction("ShellExecute",                 Sc_PluginStub_Void);
     return true;
   }
-  else if (ags_stricmp(name, "agsappopenurl") == 0)
+  else if (ags_stricmp(name, "agsappopenurl") == 0 ||
+           ags_stricmp(name, "ags_SnowRain20") == 0)
```

Comment:
I think this is a typo

Reply:
I put this new line into the wrong condition.